theme/static/style/style.css

```diff
@@ -412,6 +431,80 @@ pre code {
 @media screen and (max-width: 991px){
     /* For tablets: */
 
+    /*navigation section*/
+    nav img {
+        margin-top: 1.25rem;
+        max-width: 20rem;
```

Comment:
A max width of 70-75% works fine for me. However this image doesn't align very well with the open-menu button. Specify width by % not rem. 

If margin-top is removed it looks good on tablets and on phone display the image is top-aligned with the menu button. Not ideal but can be fixed later.